src/WpfMath.Tests/ParserTests.fs

```diff
@@ -50,6 +50,10 @@ let spacesInText() =
 let cyrillicText() =
     verifyParseResult @"\text{абв}"
 
+[<Fact>]
+let underscoreText() =
+    verifyParseResult @"\text{_}"
```

Comment:
Let's add another interesting test here: `\text{\alpha \beta \unknowncommand}`: it will show how the algorithm handles spaces and `\commands` (it should not change them actually).